drivers/sensor/sensirion/scd4x/scd4x.c

```diff
@@ -148,16 +207,14 @@ static int scd4x_init(const struct device *dev)
         return -ENODEV;
     }
 
```

Comment:
Comment format will be addressed in an other issue.

```suggestion
    // Stop periodic measurement
```